score.py

```diff
@@ -17,11 +17,15 @@ def read_file_to_list(file_in,logger):
             data_list.append(float(line))
     return data_list
 
-def main(args):
-    assert len(args.files) == 2, "Number of files specified must equal 2. "
+def main(files):
+    """
+    Evaluate a set of predictions with regard to a reference set.
+    :param files: List of paths to two text files containing predictions and references. 
+    """
+    assert len(files) == 2, "Number of files specified must equal 2. "
 
     data = []
-    for i,path in enumerate(args.files,1):
+    for i,path in enumerate(files,1):
```

Comment:
Not related to this change, but since `i` isn't used, we don't need to use `enumerate` either, and we certain don't need to pass the `, 1)` argument to enumerate to tell it to start from 1. So this can be

    for path in files:

Maybe it was used at some point?